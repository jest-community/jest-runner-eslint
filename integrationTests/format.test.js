const runJest = require('./runJest');

it('Applies custom formatter', () => {
  return expect(runJest('format')).resolves.toMatchSnapshot();
});
