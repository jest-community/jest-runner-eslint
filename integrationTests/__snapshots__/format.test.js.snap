// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Applies custom formatter 1`] = `
"FAIL integrationTests/__fixtures__/format/__eslint__/file.js
  âœ• ESLint
      at mocked-stack-trace  2 |
  3 | if (a === 2) {
> 4 |   hello();
    |   ^
  5 |   bye();
  6 | }
  7 |
mocked-stack-trace  3 | if (a === 2) {
  4 |   hello();
> 5 |   bye();
    |   ^
  6 | }
  7 |
2 errors found.
Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 total
Snapshots:   0 total
Time:
Ran all test suites.

"
`;
